src/Symfony/Component/RateLimiter/CompoundLimiter.php

```diff
@@ -25,17 +25,28 @@ final class CompoundLimiter implements LimiterInterface
      */
     public function __construct(array $limiters)
     {
+        if ([] === $limiters) {
+            throw new \LogicException("Limiters shouldn't be empty. Please specify at least one.");
```

Comment:
the message should give some more context - at least it should mention the class

Reply:
What about: 

`throw new \LogicException(sprintf("%s::%s() require at least one limiter", self::class, __METHOD__));`

?

Reply:
I like it pretty much more, I will push it later and we will see